src/scripts/components/Input.jsx

```diff
@@ -75,7 +75,7 @@ class Input extends React.Component {
     this.setState({ value: event.target.value });
 
     if (this.props.onChange && typeof (this.props.onChange === 'function')) {
-      this.props.onChange(event.target.id, event.target.value);
+      this.props.onChange(event.target.id, event.target.value.trim());
```

Comment:
Do we need to add the .trim to this.setState({ value: this.props.initialValue }); on the willmount's?